src/renderer/root/components/AuthenticatedLayout/AuthenticatedLayout.scss

```diff
@@ -74,7 +72,8 @@ $sidebar-shadow: inset -1px 0 0 #d7d8de;
         flex: 1 1 auto;
         box-sizing: border-box;
         overflow-y: auto;
-        flex: 1;
+        height: 100vh;
+        //flex: 1;
```

Comment:
You should verify if this is okay :P (Duplicate `flex 1` which resulted in the page height being fullscreen with `height: 100vh`

Reply:
Try `flex: 1 1 100%;`

Reply:
This should be sufficient:

```js
.content {
  flex: 1 1 100%;
  overflow-y: auto;
  box-sizing: border-box;
}
```

Reply:
done :P Fixed it